Updates for the statistics usability enhancement feature (predicate columns, config persistence, synchronous loading) (#7412)
pingcap/docs: system-variables.md

- Default value: `ON`
- This variable controls whether to record the execution information of each operator in the slow query log.

### tidb_enable_column_tracking <span class="version-mark">New in v5.4.0</span>

> **Warning:**
>

> > Currently, collecting statistics for `PREDICATE COLUMNS` is an experimental feature. It is not recommended that you use it in production environments.

Reviewer: ```suggestion
> Collecting statistics of `PREDICATE COLUMNS` is an experimental feature in v5.4.0 and is not recommended for production.
```

Author: ditto

- Scope: GLOBAL
- Default value: `OFF`

> - This variable controls whether to enable TiDB to collect `PREDICATE COLUMNS`. After enabling the collection, if you disable it, the information of previously collected `PREDICATE COLUMNS` is cleared. For details, see [Collect statistics for some columns](/statistics.md#).

Reviewer: ```suggestion
- This variable controls whether to enable TiDB to collect `PREDICATE COLUMNS`. After you disable this variable, the previously collected `PREDICATE COLUMNS` are cleared. For details, see [Collect statistics for some columns](/statistics.md#).
```
存在以前的 Predicate columns 就说明以前打开过这个变量，无需强调 after enabling the collection.

- Default value: ""
- This variable is used to set the time point at which the data is read by the session. For example, when you set the variable to "2017-11-11 20:20:20" or a TSO number like "400036290571534337", the current session reads the data of this moment.

### tidb_stats_load_sync_wait <span class="version-mark">New in v5.4.0</span>

> **WARNING:**
>

> > Currently, synchronously loading statistics is an experimental feature. It is not recommended that you use it in production environments.

Reviewer: ```suggestion
> Synchronously loading statistics is an experimental feature in v5.4.0 and is not recommended for production.
```

Author: ditto.

- Scope: SESSION | GLOBAL
- Default value: `0`
- Unit: milliseconds
- Range: `[0, 4294967295]`
- This variable controls whether to enable the synchronously loading statistics feature. The default value `0` means that the feature is disabled. To enable the feature, you can set this variable to a timeout (in milliseconds) that SQL optimization can wait for at most to synchronously load complete column statistics. For details, see [Load statistics](/statistics.md#load-statistics).

### `tidb_stats_load_pseudo_timeout` <span class="version-mark">New in v5.4.0</span>

> **WARNING:**
>

> > Currently, synchronously loading statistics is an experimental feature. It is not recommended that you use it in production environments.

Reviewer: ```suggestion
> Synchronously loading statistics is an experimental feature in v5.4.0 and is not recommended for production.
```

- Scope: GLOBAL
- Default value: `OFF`

> - This variable controls how TiDB behaves when the waiting time of SQL optimization reaches the timeout to synchronously load complete column statistics. The default value `OFF` means that SQL execution fails after the timeout. If you set this variable to `ON`, the SQL optimization gets back to using pseudo statistics after the timeout.

Reviewer: ```suggestion
- This variable controls how TiDB behaves when the waiting time of SQL optimization reaches the timeout to synchronously load complete column statistics. The default value `OFF` means that SQL execution fails after the timeout. If you set this variable to `ON`, the SQL optimization gets back to use pseudo statistics after the timeout.
```

Author: rejected
reason: get back to doing something  
https://idioms.thefreedictionary.com/get+back+to+doing+something#:~:text=To%20take%20up%20some%20activity,after%20some%20hiatus%20or%20interruption.
